### ISSUE TITLE
Missing preceding versions

### ISSUE THREAD

**Issue by strogonoff** [ietf-ribose/relaton-data-ids#10]
There is version 10 of this draft, but versions 1–9 are missing:

https://github.com/ietf-ribose/relaton-data-ids/blob/main/data/draft-leiba-imap-implement-guide-10.yaml

**Comment by ronaldtse** [comment]
Ha ha ha... You have a good eye... I don't have an answer 😉 

I think we will require some help here, the version number is likely not "enforced". I suspect the BibXML Service should just accept that version numbers may not always be contiguous.

@rjsparks is the proposed behavior appropriate?

**Comment by strogonoff** [comment]
Non-contiguous versions are not a problem for the service, just making sure data didn’t get lost during processing and isn’t supposed to be available

**Comment by rjsparks** [comment]
Short answer: Yes, there will be versions missing, and that's ok.

The datatracker doesn't have the metadata for some early versions of very old drafts, so it can't say who the authors were.

In this particular case, we _do_ have the older drafts in the id-archive and at some point we'll backfill the datatracker with metadata based on them. At that point the bibxml for these older versions could be pushed to the service.

In other cases, we do not have the original material in the id-archive, and will never be able to really create an accurate bibxml entry for that material.

**Comment by rjsparks** [comment]
I just checked and the older service also doesn't have bibxml for the older versions of that particular Leiba draft.

**Comment by strogonoff** [comment]
Closing (data is missing in authoritative source). For the record, @ronaldtse, I don’t think missing versions mean we should omit superseding relations (#6).
